app/assets/stylesheets/_albumInfo.scss

```diff
@@ -0,0 +1,10 @@
+ul {
+  padding: 0;
+}
+li {
+  display: block;
+  padding: 5px;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  color: #808080;
+}
```

Comment:
You've called this Sass module `albumInfo`, but you have no class names in here. Instead, you are defining these styles on _every_ `ul` & `li`, which might not be what you want. Instead, gives these class names if you want it to be specific